fontir/src/serde.rs

```diff
@@ -63,6 +63,48 @@ impl From<StaticMetadata> for StaticMetadataSerdeRepr {
     }
 }
 
+// The HashMap<NormalizedLocation, GlyphInstance> seems to throw serde for a loop sometimes
+#[derive(Serialize, Deserialize, Debug, Clone)]
+pub struct GlyphInstanceSerdeRepr {
+    location: NormalizedLocation,
+    instance: GlyphInstance,
+}
+
+#[derive(Serialize, Deserialize, Debug, Clone)]
+pub struct GlyphSerdeRepr {
+    pub name: String,
+    pub instances: Vec<GlyphInstanceSerdeRepr>,
```

Comment:
we decided to call that attribute "sources" in the fontir::ir::Glyph struct which this GlyphSerdeRepr is meant to be associated with, I don't see why use "instances" instead of the same "sources" field name